cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -12,6 +12,10 @@ spec:
     resources:
       limits:
         cpu: 100m
+        memory: 200Mi
+      requests:
+        cpu: 100m
+        memory: 200Mi
```

Comment:
1. The memory limit wouldn't actually need to scale with machine size. The more important bit is how many logs the agent can churn through per second, so we'd want to scale CPU to avoid a backlog of unprocessed logs building up. I think that'd be great. Do we have info on how much CPU a node has in salt while the node is starting up? It's a little different from heapster where we want to scale with number of nodes, whereas here we'd want to scale with number of cores on the node.

2. Not yet, only what we hear from customers. But OOMs should be next to impossible given our buffer settings. The concern that we have heard about is fluentd falling behind and dropping some logs from its buffer on nodes with a bunch of pods that log heavily.